fiftyone/core/utils.py

```diff
@@ -2870,6 +2870,39 @@ def get_multiprocessing_context():
     return multiprocessing.get_context()
 
 
+def get_cpu_count():
```

Comment:
@swheaton is proposing to add a `get_cpu_count()` utility in https://github.com/voxel51/fiftyone/pull/6290, so these two implementations should be reconciled and we should ship a single thing that does whatever combo of things we consider best practice.

Reply:
Oh thanks for calling that out. Didn't know Stuart had another implementation.